nss-day-cohort-39/github-workflow-aquarium-cohort-39-tuna
# Added angler fish + dialog to index.

### Changes
- Added Dave the angler fish to the index.
- Added dialog with details for Dave.

### Testing
- Confirm Dave is showing up on main.
- Confirm Dave's dialog and details are working (open and closing).
